demisto/demisto-sdk
# fix the print color logger issue by wrap the placeholder {} with the …

…color tag



## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11446

## Description
# What was changed?
* Fix the issue of the color in logger

# How to tests?
* Run test modeling rule and make sure the log shows properly.


Before:
![Screenshot 2024-12-23 at 13 23 26](https://github.com/user-attachments/assets/8135694b-a6c0-4355-b5cf-20522ad56e70)
After:
![Screenshot 2024-12-24 at 9 50 15](https://github.com/user-attachments/assets/056e3138-ea21-4c33-a41d-01821df293a4)

Before:
![Screenshot 2024-12-23 at 13 23 56](https://github.com/user-attachments/assets/2a6e8fcf-6185-4e5e-a10e-8b4497d5e3f7)

After:
![Screenshot 2024-12-24 at 9 54 41](https://github.com/user-attachments/assets/e472fe9b-5b3f-4a44-b97d-bf987571f100)
